docs/about/index.md

```diff
@@ -24,3 +24,19 @@ Simply download the archive, unzip and view locally in a web browser.
 ## Source code and development
 
 Plopp is hosted and developed [on GitHub](https://github.com/scipp/plopp).
+
+## Contributing to Plopp
+
+Contributions, bug reports, and ideas are always welcome.
+If in doubt whether a feature falls within the scope of Plopp please [ask on GitHub](https://github.com/scipp/plopp/issues) before implementing functionality, to reduce the risk of rejected pull requests.
+Asking and discussing first is generally always a good idea, since our road map is not very mature at this point.
+
+## FAQ
+
+```{toctree}
+---
+hidden:
+---
+
+faq
```

Comment:
Actually the one item in the faq has been moved to the `docs/customization/subplots` notebook, so I think the `faq.ipynb` notebook should just be removed. I probably forgot to remove it when I changed docs theme.